libs/documentation/src/lib/components/popup/demos/basic/popup-basic.module.ts

```diff
@@ -3,13 +3,16 @@ import { CommonModule } from '@angular/common';
 import { PopupBasic } from './popup-basic.component';
 import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
 import { SdsPopupModule } from '@gsa-sam/components';
+import { SdsIconModule, SdsPopoverModule, SdsTooltipModule } from '../../../../../../../packages/components/src/lib/public-api';
```

Comment:
Could you use absolute pathing here instead of relative - These should all be imported from `@gsa-sam/components`